packages/components/src/components/dropdown/dropdown.tsx

```diff
@@ -99,6 +99,14 @@ export class Dropdown {
      */
     @Prop() filterable = false;
 
+
+    /**
+     * If present, it dictates placeholder for an filterable input field in the dropdown. Defaults to 'Type to filter...'.
+     *
+     * @type {string}
+     */
+    @Prop() placeholder: string;
```

Comment:
https://stenciljs.com/docs/properties#default-values

Looks like we can do 

```suggestion
    @Prop() placeholder: string = 'Type to filter...';
```

(not tested)

Reply:
Tested, works. Suggestion applied.